### DISCUSSION TITLE
PHP 8.3.0 curl problem

### DISCUSSION THREAD

**Discussion by jesusAG28** [leokhoa/laragon#624] [Q&A] [answered]
Laragon issue? PHP issue? I have downloaded it from the official website just like other versions
![image](https://github.com/leokhoa/laragon/assets/33084455/355c8c36-093a-4930-9182-e214dd41b7b4)

**Comment by RegalChivas** [comment]
try reinstall SSL, and reload apache / nginx

**Reply by tecnixindo** [reply]
It's working fine when I switch to nts version

**Comment by dustinatx** [comment, marked as answer]
Try updating Apache

https://www.kreaweb.be/laragon-update-apache/

**Reply by JeFav** [reply]
Thanks, this solution works for me ! ;)

**Reply by marquez20** [reply]
Works!!

**Reply by AmirKhan47** [reply]
Worked here is the link download and paste https://www.apachelounge.com/download/VS17/binaries/httpd-2.4.58-240131-win64-VS17.zip

**Reply by datamweb** [reply]
Thanks. Works!

**Reply by HichamAbderrahim** [reply]
Thanks, it works for me !

**Reply by w3spi5** [reply]
Perfect solution !!! 👏 👏 👏

**Reply by lutfi2599** [reply]
gacor bos ku

**Reply by carlitorweb** [reply]
Confirmed, version 2.4.59 fix the problem

**Reply by hamer1818** [reply]
It worked for php 8.4.3, thank you very much. (The version I installed on httpd is 2.4.62)

**Comment by Advento27** [comment]
@jesusAG28 
It is not working for me, I uninstalled it and installed it again, updated apache and still nothing, how did you solve it?
